gen/Project.toml

```diff
@@ -0,0 +1,6 @@
+name = "MPIgen"
+uuid = "01aa870b-fa19-4f9c-adda-70d3c70d3e08"
+
+[deps]
+Clang = "40e3b903-d033-50b4-a0cc-940c62c95e31"
```

Comment:
Can you please add the compat for Clang?

Reply:
Yes, but please note that this will that we need https://github.com/JuliaInterop/Clang.jl/pull/385, so the current compat entry for `Clang` is a lie.

Reply:
Fixed by https://github.com/JuliaRegistries/General/pull/69201.